cmd/main.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
In other projects, we've been doing `project/command_name` instead of `project/cmd` such that multiple commands can potentially be built. For example, `github.com/zmap/zdns/zdns`. 

What are your thoughts on doing `github.com/zmap/zlint/zlint`? If this is going against golang accepted practice, we should change the other ones, but it'd be nice to be consistent across Z* projects.

Reply:
One other follow up question (if you know off the top of your head), when you `go get github.com/zmap/zlint/cmd`, what's the name of the command that's installed in`bin/`?

Reply:
Most of the examples I'm seeing online are of the form `cmd/name`. If we'd like the `cmd` to be present, I think it'd be best to be doing `cmd/zlint`.

Reply:
Done.